presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -48,12 +48,45 @@ Next, create a ``config.properties`` file:
     test.application-name=verifier-test
     test-id=1
 
+Verifier can run in either ``query-bank`` or ``control-test`` mode by setting configuration
+``running-mode``.
+* ``control-test``: this is the default mode. Both the control query and the test query are
+  executed and their result checksums are compared.
```

Comment:
```suggestion
  executed and their result checksums are compared.
  
```

Reply:
@steveburnett Fixed. Thanks for reviewing!